packages/payload/src/admin/components/utilities/Preferences/index.tsx

```diff
@@ -7,7 +8,7 @@ import { useConfig } from '../Config'
 
 type PreferencesContext = {
   getPreference: <T = any>(key: string) => Promise<T> | T
-  setPreference: <T = any>(key: string, value: T) => Promise<void>
+  setPreference: <T = any>(key: string, value: T, merge?: boolean) => Promise<void>
```

Comment:
Should we change `merge` to `batch`?

Reply:
I left it `merge` since the flag is set to change the behaviour of what is being stored. The batching part is just a performance enhancement that you don't really need to know about when writing code that calls this function.